src/config.xml

```diff
@@ -1737,6 +1737,15 @@ to disable this feature.
 
   @note The availability of this option depends on whether or not doxygen
   was generated with the `-Duse_libclang=ON` option for CMake.
+]]>
+      </docs>
+    </option>
+    <option type='bool' id='CLANG_INCLUDE_INPUT_PATHS' setting='USE_LIBCLANG' defval='1'>
+      <docs>
+<![CDATA[
+  If clang assisted parsing is enabled and the \c CLANG_INCLUDE_INPUT_PATHS tag
+  is set to \c YES then doxygen will use add the directory of each input to the
+  include path.
```

Comment:
I think this sentence is not correctly formulated (especially the "use add").

Reply:
Fixed.